environment.yml

```diff
@@ -14,6 +14,18 @@ dependencies:
   - scipy>=1.0,<1.13
   - yamale>=4.0
   - h5py>=3.5
-  - s1reader>=0.2.0
+  - s1reader>=0.2.1
   - shapely>=1.8
   - requests>=2.0
+  - scikit-image
+# CI unit test
+  - pytest
+  - pluggy
+  - pytest-order
+# Dependencies for RAiDER
+  - progressbar
+  - rasterio
```

Comment:
you've started to use `rasterio` in the COMPASS code here https://github.com/opera-adt/COMPASS/blob/main/src/compass/s1_cslc_qa.py#L14-L15

(As a side note- if the test for the PR that added that is passing... does that mean we should add some [smoke test](https://en.wikipedia.org/wiki/Smoke_testing_(software)) where we're actually building and importing everything? Not sure how that one could have passed)

Reply:
we also now are importing `geopandas` but don't have it listed here

Reply:
Unfortunately, RAiDER still needs them. `geopandas` is not directly required by RaiDER, but I've found that installing the RAiDER dependencies (i.e. `progressbar`, `rasterio`, `dem_stitcher`, and `gxx_linux-64`) resulted in installing `geopandas`.

Reply:
My bad - I missed the `RAiDER` comment above